src/isar/storage/slimm_storage.py

```diff
@@ -30,41 +31,61 @@ def __init__(self, request_handler: RequestHandler) -> None:
 
         self.url: str = settings.SLIMM_API_URL
 
-    def store(self, inspection: Inspection, metadata: MissionMetadata) -> None:
-        token: str = self.credentials.get_token(self.request_scope).token
-
-        request_url: str = f"{self.url}/UploadSingleFile"
-
-        inspection_type: str = get_inspection_type(inspection=inspection)
+    def store(self, inspection: Inspection, metadata: MissionMetadata):
         filename: str = get_filename(
             mission_id=metadata.mission_id,
-            inspection_type=inspection_type,
+            inspection_type=type(inspection).__name__,
             inspection_id=inspection.id,
         )
-        filename_with_ending: str = f"{filename}.{inspection.metadata.file_type}"
+        filename = f"{filename}.{inspection.metadata.file_type}"
```

Comment:
typehinting missing (: str)

Reply:
Nope, because `filename` is defined on line 35, we are just reusing the variable and hence do not need to typehint it again